pages/lobby.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import Layout from '../components/Layout';
-import NameplateGroup, {NameplateProps} from '../components/NameplateGroup';
+import NameplateGroup, {NameplateGroupProps} from '../components/nameplate/NameplateGroup';
 import PlaylistVisual from '../components/PlaylistVisual';
 import styles from '../styles/lobby.module.scss';
 
-const USERS: NameplateProps[] = [
+const USERS: NameplateGroupProps['names'] = [
```

Comment:
lmao am i trippin what is the `'names'` part of this

Reply:
Was trying to save on an type import from an otherwise unused component, but yeah not worth the confusion